modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java

```diff
@@ -188,8 +188,11 @@ public void deleteQueue(String queueName) throws AndesException {
         MessagingEngine.getInstance().purgeMessages(queueName, null, false);
 
         //delete all subscription entries if remaining (inactive entries)
+        ClusterResourceHolder.getInstance().getSubscriptionManager().deleteAllLocalSubscriptionsOfBoundQueue(queueName);
+
```

Comment:
Better to store "ClusterResourceHolder.getInstance().getSubscriptionManager()" in a field variable since we are reusing it.